inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape_broadcast.cpp

```diff
@@ -33,8 +33,12 @@ void dynamicToStaticShapeBroadcast(std::shared_ptr<ngraph::Node> target) {
         staticShapeBroadcast = std::make_shared<ngraph::vpu::op::StaticShapeBroadcast>(
                 broadcast->input_value(0),
                 broadcast->input_value(1));
+    } else if (broadcast->get_broadcast_spec() == ngraph::op::BroadcastType::BIDIRECTIONAL) {
```

Comment:
I guess all these changes in DTS may be reverted, since they will be done and tested separately.